templates/controllers/tab/settings/productionStage/form/productionStageForm.tpl

```diff
@@ -46,4 +44,3 @@
 		{fbvFormButtons id="productionStageFormSubmit" submitText="common.save" hideCancel=true}
 	{/if}
 </form>
-<p><span class="formRequired">{translate key="common.requiredField"}</span></p>
```

Comment:
Hmm, that contradicts the help text at the top of the form. Also, I can submit the form without those fields and there's no error. Maybe this setting has been changed?